### PR TITLE
Add Maryland EJ Screen into data-pipeline's ETL and provide automated scoring and statistics outputs

### PR DESCRIPTION
**Pull request by saran-ahluwalia**
Addresses #239 

To replicate: 

1. `poetry run python3 data_pipeline/application.py etl-run --dataset mdejscreen`
2. `poetry run python3 data_pipeline/application.py score-run`
3. In the comparison tool Jupyter notebook run the entire notebook to generate scores for Maryland based on the 90th percentile

Outstanding questions:

1. Is the level of detail and comments all necessary and clear?
2. Is it required to include the output Maryland basic statistics for burdened tracts?
3. The threshold for prioritized communities that is used here is the 90th percentile. Is this the best threshold to use?  The literature from MD EJ Screen doesn't offer too much guidance.

### CODE REVIEW

File: data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb [hunk 1]
```diff
@@ -306,6 +329,15 @@
...
     "    merged_df[ExtractTransformLoad.GEOID_TRACT_FIELD_NAME].str.len().unique()\n",
     ")\n",
+    "# Debugging: Uncomment lines 24-29 to identify which portions of the dataset\n",
+    "# are not coherent with the scoring comparison's contract\n",
+    "# mask = (merged_df[\n",
+    "# ExtractTransformLoad.GEOID_TRACT_FIELD_NAME].str.len() != 11)\n",
+    "# merged_df = merged_df.loc[mask] \n",
+    "# print(merged_df)\n",
+    "\n",
+    "\n",
```
Comment (saran-ahluwalia) [inline data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb:339]: I added these commented-out lines in the event someone needs to identify which tracts violate the length.

File: data/data-pipeline/data_pipeline/etl/sources/mdejscreen/etl.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                "EJScore": self.MDEJSCREEN_PERCENTILE_FIELD_NAME,
+            },
+            inplace=True,
+        )
+
+        # Baseline Comparisons at quartiles and the 90th percentile
+        # Interpretation: The score is greater than
+        # N% of the tracts in the state
+        self.df[field_names.MDEjSCREEN_TRACT_25_PERCENT_FIELD] = (
+            self.df[self.MDEJSCREEN_PERCENTILE_FIELD_NAME] > 0.25
```
Comment (saran-ahluwalia) [inline data/data-pipeline/data_pipeline/etl/sources/mdejscreen/etl.py:94]: I added these for future analyses and for comparisons with other measures used in Census tract datasets.

Comment (lucasmbrown-usds) [reply data/data-pipeline/data_pipeline/etl/sources/mdejscreen/etl.py:96]: Does MD EJSCREEN team never use a particular cutoff anywhere for this? Is there a reason you picked 25% here?

**Comment by lucasmbrown-usds** [comment]
Test.

File: data/data-pipeline/data_pipeline/etl/sources/mdejscreen/etl.py (hunk 2 above)
Comment (saran-ahluwalia) [reply data/data-pipeline/data_pipeline/etl/sources/mdejscreen/etl.py:96]: I will re-review the associated papers and the original white paper again, and reconfirm using the associated [paper](https://www.mdpi.com/1660-4601/16/3/348/htm) that we had discussed earlier this month. 

On cursory glance with an example - citing Figure 4 from the aforementioned paper (below) - there appears to be an interval of values that are produced (albeit from the scoring perspective):

image

Again, I will re-review the reference and the associated references to double check. In addition, is it all possible for us to receive feedback from Dr. Wilson on a reasonable threshold for burden (assuming the literature review doesn't produce anything definitive)?

**Comment by saran-ahluwalia** [comment]
@lucasmbrown-usds Referencing [this](https://github.com/usds/justice40-tool/pull/1067#issuecomment-1002837030). I wholeheartedly agree with you that this may be the cause for such errors. What I can do is just close this PR and reopen a new one using the main repository for both local and for origin. How does that sound?

**Comment by lucasmbrown-usds** [comment]
> @lucasmbrown-usds Referencing [this](https://github.com/usds/justice40-tool/pull/1067#issuecomment-1002837030). I wholeheartedly agree with you that this may be the cause for such errors. What I can do is just close this PR and reopen a new one using the main repository for both local and for origin. How does that sound?

That might work! Are you able to appropriately copy over your commits to make sure we don't lose them? thanks!

**Comment by saran-ahluwalia** [comment]
@lucasmbrown-usds I will start off of a fresh branch off main and just copy over the appropriate committed files.

**Comment by saran-ahluwalia** [comment]
Closing PR and reopening a topic branch in order to ensure @lucasmbrown-usds can review

**Pull request closed without merging**